Stamen maps for spatial data

SwarnaBharathiMantena opened the pull request:
edav community contribution - Neelam Patodia, Kumari Nishu

SwarnaBharathiMantena committed:
file uploaded

jtr13 commented on Date_and_Time_Cheatsheet.Rmd:
```diff
@@ -0,0 +1,32 @@
+Kanika Aggarwal and Swarna Bharathi Mantena
+
+We both prepared a cheatsheet for the Date and Time manipulations in R programming language. Mentioned below is the github link and the references used. 
+NOTE: We uploaded the chestsheet as a pdf file in the github repository. 
```
typo in cheatsheet ... please fix

jtr13 commented on Stamen_Maps_for_Spatial_Data.Rmd:
````diff
@@ -0,0 +1,135 @@
+We conduct a comparative study between different types of graphs to understand the visualization tool which best demonstrates spatial data.We use the publicly available data on the number of vehicle collisions in New York for this demonstration.   
+
+
+```{r setup, include=FALSE}
````
Please remove "setup" label from this chunk

jtr13 commented on Stamen_Maps_for_Spatial_Data.Rmd:
```diff
@@ -0,0 +1,135 @@
+library(gridExtra)
+
+#Loading the collisions dataset available from "https://data.cityofnewyork.us/Public-Safety/NYPD-Motor-Vehicle-Collisions-Crashes/h9gi-nx95/data"
+raw_df<-read.csv(file="/Users/neelampatodia/Downloads/collisions_data.csv", header=TRUE, sep=",")
```
Please add `collisions_data.csv` to the resources folder and then change this link to that location.

SwarnaBharathiMantena replied:
A sample of the collisions data has been added to the resources and the link updated.

jtr13 commented:
Please make the changes indicated above... thanks!

SwarnaBharathiMantena committed:
Update Stamen_Maps_for_Spatial_Data.Rmd

SwarnaBharathiMantena commented:
Hello Professor,

We are using an original NYC collisions data which is of 355MB. As such as we had discussed with you after class, instead of uploading this file under resources, we thought of citing the original data source itself. Should we continue with this procedure or should we upload the data?
 Kindly suggest. 

Thanks,
Group 35

jtr13 commented:
We discussed a few options after class: 1) moving the analysis to your repo, 2) sampling to shrink the dataset, 3) read the data from Google drive.

SwarnaBharathiMantena committed:
group 35 data uploaded2

SwarnaBharathiMantena committed:
Update Stamen_Maps_for_Spatial_Data.Rmd

SwarnaBharathiMantena committed:
Update Stamen_Maps_for_Spatial_Data.Rmd

jtr13 commented:
Almost there... please fix the link where you read the data so that it reads from this repo, not yours. Thanks.

SwarnaBharathiMantena committed:
Update Stamen_Maps_for_Spatial_Data.Rmd

SwarnaBharathiMantena commented:
Hello Professor,

The change has been made as suggested. Thank you.

jtr13 commented:
Great, thank you.

jtr13 committed:
Update _bookdown.yml

jtr13 committed:
sync

Merge remote-tracking branch 'refs/remotes/origin/master'

Conflicts:
	_bookdown.yml

jtr13 committed:
edits to Stamen maps chapter

jtr13 committed:
sync with PR

Merge remote-tracking branch 'refs/remotes/origin/master'

Conflicts:
	_bookdown.yml

jtr13 committed:
read data from source, other minor edits

jtr13 committed:
reorder bars

jtr13 committed:
remove .desc = FALSE

jtr13 commented:
I see what you mean about the sample not working well, so I changed the code to read the `.csv` directly from the URL. Also made some minor edits.

jtr13 commented:
🎉 Your pull request has been merged into master. Thanks for your contribution!
